build.yml

```diff
@@ -0,0 +1,27 @@
+sdk: '2'
+language: bash
+go-runtime:
+  version: '1.10-alpine'
+native:
+  image: 'openjdk:8-jre-alpine'
+  static:
+      - path: src/main/sh/native.sh
+        dest: native
+  deps:
+      - 'apk add gradle'
+  build:
+    image: 'openjdk:8-slim'
+    deps:
+      - 'apt-get update && apt-get install -y make bash wget gradle'
+    run:
+      - 'make'
+      - 'gradle shadowJar'
+    artifacts:
+      - path: '/native/build/libs/native-jar-with-dependencies.jar'
+  test:
+    run:
+      # download and use gradle 4.9 since alpine's version is bugged for tests
```

Comment:
Tests are running on top of `openjdk:8-slim`, according to the config

Reply:
Interesting. Why are they using that if it's the image specificed for "build" that is at the same level as "test"? Adding an `image: 'openjdk:8-jre-alpine'` inside `test` will fix it?

Reply:
This comment is still not addressed. It tells that "gradle from alpine is broken", but an image that runs tests is not Alpine, it's Debian. Are you sure you still need to download manually grade instead of using a version provided by Debian, or by some PPA?

Reply:
Ahh, ok, I misunderstood your comment as saying that I shouldn't run that base image for the tests. I'll take a look at this.

Reply:
PR updated.